uuid_cache_so.csv

```diff
@@ -13023,6 +13023,7506 @@ pkg-soic127p762x270-8-text-name-density~c,6706e38b-e92c-4458-908e-b83edcd2dcd3
+pkg-ssop24p40_500x640x200l100x23-lead-proj-4,8fff8897-28ca-486a-8c3a-879b30c52384
+pkg-ssop24p40_500x640x200l100x23-lead-proj-5,b41f452f-6508-4fde-81b5-07e117061442
+pkg-ssop24p40_500x640x200l100x23-lead-proj-6,540a692f-46b2-44bd-807f-ee2e755a4e79
+pkg-ssop24p40_500x640x200l100x23-lead-proj-7,5c9d29f7-5328-4803-a8eb-d16b4342f7d9
+pkg-ssop24p40_500x640x200l100x23-lead-proj-8,c83de060-5fb9-430b-b3f3-fdb0e49b5dee
+pkg-ssop24p40_500x640x200l100x23-lead-proj-9,02516904-1127-4eea-89c4-b23b7c98c979
+pkg-ssop24p40_500x640x200l100x23-pad-1,f803c821-3f89-4617-9eb7-c23e74ea27ce
+pkg-ssop24p40_500x640x200l100x23-pad-10,b7ef7fa2-a25b-463e-b83f-10f9d2d10783
+pkg-ssop24p40_500x640x200l100x23-pad-11,4490eaf9-21b2-4927-a5aa-5afdfa6540ba
+pkg-ssop24p40_500x640x200l100x23-pad-12,1ac3e71a-35a4-41c6-8e36-9358afa323d8
+pkg-ssop24p40_500x640x200l100x23-pad-13,d364c421-c267-4998-9ac7-0ac6ef9f6253
+pkg-ssop24p40_500x640x200l100x23-pad-14,cb305446-92cd-4c53-b2ac-debd63628f62
+pkg-ssop24p40_500x640x200l100x23-pad-15,c6987b01-2e86-4425-bccb-18bedc28ea1d
+pkg-ssop24p40_500x640x200l100x23-pad-16,ba9383ed-6e32-45e1-b9a5-8bd69ce95e69
+pkg-ssop24p40_500x640x200l100x23-pad-17,00d44464-4cb5-4f37-9400-f7ceccfc3a42
+pkg-ssop24p40_500x640x200l100x23-pad-18,ac5768c5-a212-436b-a37c-a65c566f8f72
+pkg-ssop24p40_500x640x200l100x23-pad-19,30f6ace2-6f42-4038-b4fb-aba0488e89e8
+pkg-ssop24p40_500x640x200l100x23-pad-2,c8fac7cf-564d-46d6-b58e-cc210e697a2b
+pkg-ssop24p40_500x640x200l100x23-pad-20,bb19eb97-ae49-42c7-b649-4dfd892d9db5
+pkg-ssop24p40_500x640x200l100x23-pad-21,b7581ba9-13de-4034-8759-a2e2d21ddb3f
+pkg-ssop24p40_500x640x200l100x23-pad-22,33a32388-fbc7-455b-be22-e8510794ef84
+pkg-ssop24p40_500x640x200l100x23-pad-23,20cb879d-4d7c-4cb0-8cb2-a82c04508699
+pkg-ssop24p40_500x640x200l100x23-pad-24,88fdc3ac-736e-4d58-bbb6-d7104fa6ab13
+pkg-ssop24p40_500x640x200l100x23-pad-3,2759c63c-9f08-46df-8ce6-ae75310497d2
+pkg-ssop24p40_500x640x200l100x23-pad-4,3251ba5b-ab89-4a1d-a15b-1d05b84d24c6
+pkg-ssop24p40_500x640x200l100x23-pad-5,aa8f7ad5-ad22-4adf-bcee-b89eddbd5d2a
+pkg-ssop24p40_500x640x200l100x23-pad-6,6020b6aa-4d0c-482b-aaea-7b375bf35cdd
+pkg-ssop24p40_500x640x200l100x23-pad-7,f089997b-4c2a-4dc9-9236-1f14954d684e
+pkg-ssop24p40_500x640x200l100x23-pad-8,8320c65d-f90e-45f3-989a-715414a807d3
+pkg-ssop24p40_500x640x200l100x23-pad-9,3949373f-a86a-4e4c-9438-39e2d85a2960
+pkg-ssop24p40_500x640x200l100x23-pkg,dfcd74f6-c903-43eb-b6ee-63d1dbccb6c7
+pkg-ssop24p40_500x640x200l100x23-polygon-courtyard-density~a,b7734861-9996-41ac-b36f-c3b9e239463b
+pkg-ssop24p40_500x640x200l100x23-polygon-courtyard-density~b,e3fd83db-9af3-4d4f-9a76-48e1e43bf94c
+pkg-ssop24p40_500x640x200l100x23-polygon-courtyard-density~c,6c3ac7b3-6486-4c2e-bac5-599d8e48bfdb
+pkg-ssop24p40_500x640x200l100x23-polygon-outline-density~a,1f8d0783-f7e4-4493-ad6f-ef281a2e426a
+pkg-ssop24p40_500x640x200l100x23-polygon-outline-density~b,56588f58-5a2d-4150-90a3-0e2eb5ca140e
+pkg-ssop
```

Comment:
Oops, I missed to change the most important one of the UUIDs, sorry! :sob:

```suggestion
pkg-ssop28p65_1020x780x200l125x38-pkg,2bdb4572-0418-4f98-8cce-1e128f01afd4
```

Reply:
Fixed.